src/main/java/au/gov/ga/geodesy/port/adapter/geodesyml/StringAdapter.java

```diff
@@ -0,0 +1,29 @@
+package au.gov.ga.geodesy.port.adapter.geodesyml;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+/**
+ * Attempt to filter out certain values like "N/A", "(enter antenna type here)", and "".
+ */
```

Comment:
N/A, (), "" are issues with sopac sitelogs, not so far with GeodesyML. I think we should stick to trimming until we run into other issues.